packages/apps/human-app/frontend/src/pages/worker/jobs/components/my-jobs/desktop/my-jobs-table.tsx

```diff
@@ -202,10 +204,14 @@ const getColumnsDefinition = ({
                 {t('worker.jobs.solve')}
               </TableButton>
               <RejectButton
-                disabled={buttonDisabled}
+                disabled={buttonDisabled || isPending}
```

Comment:
Probably from UX perspective it also makes sense to disable "Solve" button while this one is pending